src/python/engagement_edge/tests/test_engagement_edge.py

```diff
@@ -8,19 +8,9 @@
 # gross hack because engagement edge is pseudo singleton
 JWT_SECRET.secret = "hey im a fake secret"
 
-
-class TestEngagementEdgeChalice(unittest.TestCase):
```

Comment:
imo it's a worthy test to have. As long as it's on our plate to resurrect I'm okay with removing it.

(And I actually think that early bailing out of the program if all env vars aren't specified might actually be desirable)

Reply:
I agree that bailing out of the _program_ if an environment variable isn't present is a great idea. The problem here is that when this test file is read in during pytest's collection phase, the fact that the environment variable isn't present causes _that_ to throw an exception. It doesn't even get to the point where it can _run_ the tests.

The problem with this particular test is that passing actually required `IS_LOCAL` to be set, so it was really testing _that_ logic, rather than anything real.

That being said, I'll file a ticket to re-create a test that actually exercises the true logic.

Reply:
Issue added: https://github.com/grapl-security/issue-tracker/issues/244